Lib/urllib/parse.py

```diff
@@ -28,6 +28,7 @@
 """
 
 from collections import namedtuple
+from enum import Flag, auto
```

Comment:
`import enum` and refer to them as `enum.Flag` and `enum.auto`.

Reply:
fixed 0ec4a4edf6